doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -714,29 +714,99 @@ general for HDDs, the bluestore throttle values are expected to be higher when
 compared to SSDs.
 
 
-Specifying  Max OSD Capacity
-----------------------------
+Override Max IOPS Capacity of an OSD
+------------------------------------
 
-The steps in this section may be performed only if you want to override the
-max osd capacity automatically set during OSD initialization. The option
-``osd_mclock_max_capacity_iops_[hdd, ssd]`` for an OSD can be set by running the
-following command:
+The steps in this section may be performed to override the max IOPS capacity of
```

Comment:
to set or override

works just fine even if the osd hasn't had it set automatically.

Reply:
Done. Changed the title of the section and edited the content a bit.